src/libra_py/workflows/nbra/step3.py

```diff
@@ -454,11 +452,20 @@ def run(params):
         
         # Output the resulting Hamiltonians
         for i in xrange(nsteps):
+
             re_filename = prms["output_set_paths"][idata] + prms["Hvib_re_prefix"] + str(i) + prms["Hvib_re_suffix"]
             im_filename = prms["output_set_paths"][idata] + prms["Hvib_im_prefix"] + str(i) + prms["Hvib_im_suffix"]        
+
             Hvib[i].real().show_matrix(re_filename)
             Hvib[i].imag().show_matrix(im_filename)
 
+            # Make time-derivative overlap matriices for chi basis and print
```

Comment:
this section should not be here. Remember, there should be no reference to any specific basis in this script. If one needs those elements - this should be done in a separate function/module